course-v2/layouts/partials/resources_header.html

```diff
@@ -22,6 +22,11 @@ <h2>Download</h2>
       </div>
     </div>
   </div>
+  <div class="alert-warning p-2 touch-device-only">
+    <strong>Note: </strong>
+    The downloaded course may not work on mobile devices.
+    We recommend using a desktop computer.
```

Comment:
```suggestion
    We recommend using a computer.
```

Someone noted that it works fine on a laptop, so we can remove "desktop"